Source/WebCore/css/parser/SizesCalcParser.cpp

```diff
@@ -62,24 +62,19 @@ static bool operatorPriority(UChar cc, bool& highPriority)
 
 bool SizesCalcParser::handleOperator(Vector<CSSParserToken>& stack, const CSSParserToken& token)
 {
-    // If the token is an operator, o1, then:
```

Comment:
Can you add `Merge: https://chromium.googlesource.com/chromium/src.git/+/e0bcd910832c55445bc6bb4156acd716e0667248` in commit and also update comments similar to Blink's commit? Thanks for tackling this.